### ISSUE TITLE
Letters are reversed

### ISSUE THREAD

**Issue by thainguyenit** [bensonruan/webcam-easy#13]
Hi,
Please help me, how to fix a letters are reversed bug after snap a picture.
Thank you.

**Comment by novelnet** [comment]
Set facingMode to 'environment' and override getVideoInputs like this works for me:

```javascript
import Webcam from 'webcam-easy';
import b64toBlob from 'b64-to-blob';

class NoMirrorWebcam extends Webcam {
    // override function as facingMode is set to 'user' when webcamList == 1 -> facingMode decides mirroring
    getVideoInputs(mediaDevices) {
        const webcamList = super.getVideoInputs(mediaDevices)
        this._facingMode = 'environment';
        return webcamList;
    }
}
```

**Comment by jayanthymohit6** [comment]
hey @novelnet  where should I place this block to fix the issue ?   I have the webcam-easy.js which has the WebCam class .And in my code I initialize the object like this :
const webcam = new Webcam(webcamElement, 'environment', canvasElement, snapSoundElement);

thanks.

**Comment by Jarff** [comment]
> hey @novelnet where should I place this block to fix the issue ? I have the webcam-easy.js which has the WebCam class .And in my code I initialize the object like this : const webcam = new Webcam(webcamElement, 'environment', canvasElement, snapSoundElement);
> 
> thanks.

Yeah, I can confirm that the suggestion is working. One option is to that class and export it, I like to do it by creating an index.js inside a Folder with the same name as the class has.

`
import Webcam from 'webcam-easy';
//import b64toBlob from 'b64-to-blob'; I commented it out because I'm not using it

class NoMirrorWebcam extends Webcam {
    // override function as facingMode is set to 'user' when webcamList == 1 -> facingMode decides mirroring
    getVideoInputs(mediaDevices) {
        const webcamList = super.getVideoInputs(mediaDevices)
        this._facingMode = 'environment';
        return webcamList;
    }
}

export default NoMirrorWebcam`

Then in the script you are using the webcam you should import your class instead of the installed on rpm,

`import NoMirrorWebcam from './path/to/the/class/NoMirrorWebCam'`

Instead of using **Webcam** you should use **NoMirrorWebcam**
